purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -779,10 +780,7 @@ internal class BillingWrapper(
 
             getPurchaseType(purchase.purchaseToken) { type ->
                 completion(
-                    purchase.toStoreTransaction(
-                        type,
-                        context?.presentedOfferingId,
-                    ),
+                    purchase.toStoreTransaction(type),
```

Comment:
I might have missed this but this no need any presented offering information anymore? 🤔

Reply:
This wasn't needed since it would only enter here if `context` was null. From a few lines above (line 774), you can see that if context.productType is not null, it will return, and `productType` is not null inside the context, so it would only continue here if context was null (it was a warning from Android Studio)